_posts/en/newsletters/2023-01-25-newsletter.md

```diff
@@ -0,0 +1,167 @@
+    capture most of the benefits of `SIGHASH_GROUP` and provides the
+    significant advantage of being much easier to get into
+    production than the `SIGHASH_GROUP` soft fork consensus change.
+
+- **Request for proof that an async payment was accepted:** Valentine
+  Wallace [posted][wallace pop] to the Lightning-Dev mailing list a
+  request for researchers to investigate how someone making an [async
+  payment][topic async payments] could receive proof that they paid.
+  For traditional LN payments, the receiver generates a secret which
+  gets digested by a hash function; that digest is given to the spender
+  in a signed invoice; the spender uses an [HTLC][topic htlc] to pay
+  anyone who discloses the original secret.  That disclosed secret
+  proves the spender paid the digest contained in the signed invoice.
+
+    By contrast, async payments are accepted when the receiver is
+    offline, so they can't reveal a secret, preventing the creation of a
+    proof of payment in the current LN model.  Wallace asks researchers
+    to consider investigating how proof of payment for async payments
+    could be obtained, either in LN's current HTLC-based system or a
+    future upgrade to [PTLCs][topic ptlc].
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Bitcoin Core signing keys were removed from repo. What is the new process?]({{bse}}116649)
+  Andrew Chow explains that while the signing keys were [removed][remove builder
+  keys] from the Bitcoin Core repository, there is now a list of keys on the
+  [guix.sigs repository][guix.sigs repo] which houses [guix][topic reproducible
+  builds] build attestations.
+
+- [Why doesn't signet use a unique bech32 prefix?]({{bse}}116630)
+  Casey Rodarmor wonders why both testnet and [signet][topic signet] both use
+  the `tb1` [address prefix][wiki address prefixes]. Kalle, one of the [BIP325][] authors, explains that
+  while signet initially used a different address prefix, it was thought using the same
+  prefix would simplify usage of that alternate test network.
+
+- [Arbitrary data storage in witness?]({{bse}}116875)
+  RedGrittyBrick points to [one of several][large witness tx] recent P2TR
+  transactions containing a large amount of witness data. Other users point out
+  that the Ordinals project provides a service for including arbitrary data,
+  like [the image][ordinals example] in the transaction above, into a Bitcoin transaction using the witness.
```

Comment:
I didn’t think the response was notable (as far as sipa answers go anyways!), personally. Other than noting it was a P2TR, which I did add for context. Which part do you suggest?

Reply:
Okay; I thought you might not be aware that he had answered, since it was so recent.

I thought it was interesting (and new to me) that P2TR spends' standardness rules are "much more relaxed" than (at least) segwit v0 rules. But now that I read his answer again, he is not sure if the transactions the question is about are considered standard or not. So you're right, not worth mentioning here. 

I also thought that the summaries here _always_ include at least something from the answers given, not just the question. But there's no reason we can't just highlight a question and let readers follow the link if they want to know more. So, in short, never mind, it's okay as is.

Reply:
Yeah, there isnt a great answer yet on that one but I thought it was interesting.

Also found per their docs: 

> Inscription content is included in transaction witnesses, which receive the witness discount. To calculate the approximate fee that in inscribe transaction will pay, divide the content size by four and muliply by the fee rate.

> Inscription transactions must be less than 400,000 weight units, or they will not be relayed by Bitcoin Core. One byte of inscription content costs one weight unit. Since an inscription transaction includes not just the inscription content, limit inscription content to less than 400,000 weight units. 390,000 weight units should be safe.

https://docs.ordinals.com/guides/inscriptions.html

So it sounds like it is relayed under 400,000wu